device/src/async_device/radio.rs

```diff
@@ -29,6 +29,13 @@ pub trait PhyRxTx: Sized {
     #[cfg(not(feature = "defmt"))]
     type PhyError;
 
+    /// The antenna gain and board loss in dBi.
+    const ANTENNA_GAIN: i8 = 0;
+
+    /// The max power the radio can be instructed to transmit at. When preparing an instruction
```

Comment:
Something like this? (Also if accepted then same comments should be used for non-async radio)
```rust
/// Maximum power the radio is able to deliver (in dBm?).
/// When preparing instructions for radio, the value of maximum power will be used as an upper bound.
```

Reply:
I agree - thanks